src/utils/create-react-container.jsx

```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { createRoot } from 'react-dom/client';
+
+export function createReactContainer(
```

Comment:
why function? My default would have been to use arrow function wherever possible...